pyproject.toml

```diff
@@ -46,6 +46,12 @@ dependencies = [
 gpu = [
     "torch==2.6.0+cu124",
 ]
+rocm = [
+    "torch @ https://repo.radeon.com/rocm/manylinux/rocm-rel-6.4/torch-2.6.0%2Brocm6.4.0.git2fb0ac2b-cp310-cp310-linux_x86_64.whl",
```

Comment:
Note: These are official wheel builds by ROCm. You need to explicitly install these by running `uv sync --extra rocm`